### PR TITLE
delete invalid test case

### PR DESCRIPTION
**Pull request by honnix**
# Hey, I just made a Pull Request!

## Description

This test case is not valid anymore after https://github.com/spotify/styx/pull/651

## Checklist for PR author(s)

- [x] Changes are covered by unit test
- [x] All tests pass
- [x] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [x] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

File: styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java [hunk 1]
```diff
@@ -75,20 +74,6 @@ public void setUp() throws Exception {
         a.<TransactionFunction>getArgument(0).apply(transaction));
   }
 
-  @Test
-  public void shouldStoreNewWorkflowAndUpdateNextNaturalTrigger() throws IOException, WorkflowInitializationException {
```
Comment (danielnorberg) [inline styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java]: Or update it to match current behavior?

Comment (honnix) [reply styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java]: You already added one when working on the PR.

Comment (danielnorberg) [reply styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java]: Ok

**Comment by danielnorberg** [comment]
👍

**Pull request merged**